drafts/25-xxx-macro-expansion.txt

```diff
@@ -241,27 +345,84 @@ EXAMPLE lbe01:
 
 2.1.3 Line breaks and continuations in macro invocations (fixed-form)
 ---------------------------------------------------------------------
+[Straw poll 1, option 3 -> remove section]
```

Comment:
No I thought we decided to keep the section regardless, but mark essentially it as "rejected".

```suggestion
[Straw poll 1, option 3 -> mark subsection 2.1.3 as "rejected"]
```